### ISSUE TITLE
Missing Talkback descriptions when receiving a call

### ISSUE THREAD

**Issue by clado** [signalapp/Signal-Android#11995]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description

This issue is a duplicate of _part of_ #11658. However, I don't think I can resolve that entire issue all at once, so I think it makes sense to break it up for tracking purposes?

### Steps to reproduce
- turn Talkback on (in System settings)
- wait to receive a call
- navigate the call screen using Talkback

**Actual result:** The "answer without video", "decline", and "answer" buttons in the call screen are announced as "unlabelled"
**Expected result:** Those three buttons are correctly announced as "answer without video button", "decline button", and "answer button".

### Screenshots
![a phone screenshot of signal receiving a call from a redacted user. the grey "answer without video", red "decline", and green "answer" buttons are visible](https://user-images.githubusercontent.com/2933589/153776338-47824510-ec06-49e3-8a69-ba6f1430ec61.png)

### Device info
**Device:** Google Pixel 2
**Android version:** Android 11 (LineageOS 18.1)
**Signal version:** 5.32.4

### Link to debug log
don't wanna 😛 <3
